### PR TITLE
Updates : Issue Templates 🛠

### LINKED ISSUE #536: Updates : Issue Templates 🛠
## What i am suggesting to update ? 

I am suggesting to update the Issue templates , currently we are using `.md` files instead of the brand new issue forms with `.yml` files.

## Why to update it ? 

Adding an issue Template allows you to organize the issues made by folks in a much better way.
 - We can know if someone wants to write a test or maybe fix something in the website
 - The issue template forms makes writing details more efficient
 - Here's a general idea : 
 
image

image

## Can i work on it ? 

Yes, i will be glad to help an Opensource project grow more !

### PR DESCRIPTION
**Pull request by IAmTamal**
# Description

## Changes requested : 

- i have asked to add Issue templates in the repository
- There must be same fields as there already was in the `.md` files !
- I am working to close #536 

## Changes made

- 2 new `.yml` files were added
- Issue templates with the same labels, fields are added as there was previously in the `.md` files.
- Some fields are made `required` so that it is compulsory to add data to those fields !
- Blank issues are kept `true` and can be changed from `config.yml`

## Screenshots 
**The color of labels can be ignored as this is the fork version**

image
image
image

Fixes #536 

---

# Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by IAmTamal** [comment]
@kartikcho  - Do take a look, i'm done with the PR !

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: 💡 Feature request
+description: Have a new idea/feature ? Please suggest!
```
Comment (kartikcho) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Maybe change this to: "Got a suggestion for a new idea/feature?"

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    attributes:
+      label: Smartphones
+      description: Please complete the following information
+      value: |
+        - Device: [e.g. iPhone6]
+        - OS: [e.g. iOS8.1]
+        - Browser [e.g. stock browser, safari]
+        - Version [e.g. 22]
+    validations:
+      required: true
```
Comment (kartikcho) [inline .github/ISSUE_TEMPLATE/bug.yml:75]: Desktop and Smartphone inputs shouldn't be required either imo as only one or none of them might be relevant to a bug. 
What do you think?

**Review by kartikcho — APPROVED** [review]
Amazing PR @IAmTamal, just need a few changes before we can merge this. Thanks!

**Comment by kartikcho** [comment]
Ah looks like the node version of the project is finally deprecated but that is unrelated to this PR

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 2 above)
Comment (IAmTamal) [reply .github/ISSUE_TEMPLATE/bug.yml:75]: Hmm true, but what if someone opens it in the mobile and then find a bug ? 

Anyways, do let me know what you think, i can always remove a field !

Comment (kartikcho) [reply .github/ISSUE_TEMPLATE/bug.yml:75]: Oh I'm not suggesting to remove the field, only to make it not required so people can fill it if its relevant to the bug

Comment (kartikcho) [reply .github/ISSUE_TEMPLATE/bug.yml:75]: same for the desktop field

Comment (IAmTamal) [reply .github/ISSUE_TEMPLATE/bug.yml:75]: Oh , got it !

Comment (IAmTamal) [reply .github/ISSUE_TEMPLATE/bug.yml]: Done ✅

**Comment by kartikcho** [comment]
@allcontributors please add @IAmTamal for docs

**Pull request merged**